source/interprocedural_analyses/taint/taintConfiguration.ml

```diff
@@ -223,14 +223,37 @@ let parse source_jsons =
     | `String "parametric" -> AnnotationParser.Parametric
     | unexpected -> failwith (Format.sprintf "Unexpected kind %s" (Json.Util.to_string unexpected))
   in
+  let unknown_keys_count = ref 0 in
+  let check_keys ~required_keys ~valid_keys ~current_keys ~section =
+    let valid_keys_hash_set = String.Hash_set.of_list valid_keys in
+    let current_keys_hash_set = String.Hash_set.of_list current_keys in
+    let check_required_key_present key =
+      if not (Hash_set.mem current_keys_hash_set key) then
+        failwith (Format.sprintf "Required key %s is not found in section `%s`" key section)
```

Comment:
nitpick: add backticks
```suggestion
        failwith (Format.sprintf "Required key `%s` is not found in section `%s`" key section)
```

Reply:
Is backticks for values from users?

Reply:
I guess you have a point. I'm fine either way.